src/datachain/lib/file.py

```diff
@@ -122,7 +122,21 @@ def resolve(cls, file: "File", location: list[dict]):
 
 
 class File(DataModel):
-    """`DataModel` for reading binary files."""
+    """
+    `DataModel` for reading binary files.
+
+    Attributes:
+        source (str): The source of the file (e.g., 's3://bucket-name/').
```

Comment:
how meaningful are these descriptions? do they give additional information? 

also, Pydantic supports some kind of programmatic descriptions, should we use those?

Reply:
Well... Good question. It is not strictly necessary, but I've added this also because of the docs.

Before (https://docs.datachain.ai/references/file/#datachain.lib.file.File):
<img width="1791" alt="image" src="https://github.com/user-attachments/assets/7fdb94d2-f0c5-4635-9023-26d7f296150b" />

After (https://b5b85e41.datachain-documentation.pages.dev/references/data-types/file/#datachain.lib.file.File):
<img width="1790" alt="image" src="https://github.com/user-attachments/assets/2a7b2c88-3f04-4b3d-9ce0-263820b15b73" />

Reply:
sounds good!